src/saml2/response.py

```diff
@@ -63,7 +63,7 @@ def for_me(condition, myself ):
 def authn_response(conf, return_addr, outstanding_queries=None,
                     log=None, timeslack=0, debug=0, asynchop=True,
                     allow_unsolicited=False):
-    sec = security_context(conf)
+    sec = security_context(conf, log=log)
```

Comment:
whoops, was looking in the wrong branch. Here's the more accurate link since it links directly to `security_context` rather than the class it returns an instance of

https://github.com/BetterWorks/pysaml2/blob/0.4.x/src/saml2/sigver.py#L448